openstack/keystone/templates/ingress-api.yaml

```diff
@@ -36,7 +36,11 @@ data:
 ---
 {{- end }}
 {{- end }}
+{{- if .Capabilities.APIVersions.Has "apps/v1" }}
+apiVersion: apps/v1
```

Comment:
this is of `kind: Ingress`
so I suggest to use `apiVersion: networking.k8s.io/v1beta1`